src/qwed_new/probe.js

```
@@ -0,0 +1,21 @@
+// QWED probe: JS engine coverage — intentionally vulnerable.
+function renderComment(html) {
+    document.getElementById("c").innerHTML = html; // XSS sink
+}
+
+function runUser(code) {
+    return eval(code); // dynamic execution
```

Biome (2.5.3)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)